Source/VolunteerReporting/Web.Angular/src/environments/environment.ts

```diff
@@ -5,5 +5,5 @@
 
 export const environment = {
   production: false,
-  api: 'http://dev.cbsrc.org/volunteerreportingbackend'
+  api: 'http://localhost:5000'
 };
```

Comment:
It needs to be  localhost;5000, right @karolikl ?

Reply:
Correct :)